api/db-cdr/generate-cdr/generate-bigquery-cloudsql-cdr.sh

```diff
@@ -85,23 +92,46 @@ if [[ $CDR_VERSION =~ ^[0-9]{4}(0[1-9]|1[0-2])(0[1-9]|[1-2][0-9]|3[0-1])$ ]]; th
     exit 1
 fi
 
-# Make BigQuery dbs
+WORKBENCH_DATASET=cdr$CDR_VERSION
+PUBLIC_DATASET=public$CDR_VERSION
+
+## Make BigQuery dbs
 echo "Making BigQuery dataset for CloudSql cdr"
 if ./generate-cdr/make-bq-data.sh --bq-project $BQ_PROJECT --bq-dataset $BQ_DATASET --workbench-project $WORKBENCH_PROJECT --account $ACCOUNT --cdr-version $CDR_VERSION
 then
-    echo "BIG QUERY CDR Data Generated"
+    echo "BigQuery cdr data generated"
 else
-    echo "FAILED To Generate BIG QUERY Data For CDR $CDR_VERSION"
+    echo "FAILED To generate BigQuery data for cdr $CDR_VERSION"
     exit 1
 fi
 
-# Make BigQuery data dump
-dataset=cdr$CDR_VERSION
+# Make public
+echo "Making BigQuery public dataset for CloudSql cdr"
+if ./generate-cdr/make-bq-public-data.sh --workbench-project $WORKBENCH_PROJECT --workbench-dataset $WORKBENCH_DATASET --public-project $PUBLIC_PROJECT --public-dataset $PUBLIC_DATASET
+then
+    echo "BigQuery public cdr data generated"
+else
+    echo "FAILED To generate public BigQuery data for $CDR_VERSION"
+    exit 1
+fi
+
+#dump workbench cdr counts
 echo "Making big query dataset for cloudsql cdr"
-if ./generate-cdr/make-bq-data-dump.sh --dataset $dataset --project $WORKBENCH_PROJECT --account $ACCOUNT --bucket $BUCKET
+if ./generate-cdr/make-bq-data-dump.sh --dataset $WORKBENCH_DATASET --project $WORKBENCH_PROJECT --account $ACCOUNT --bucket $BUCKET
 then
-    echo "BIG QUERY CDR Data Generated"
+    echo "Data dumped"
 else
-    echo "FAILED To Generate BIG QUERY Data For CDR $CDR_VERSION"
+    echo "FAILED to dump"
     exit 1
 fi
+
+# dump public counts
+dataset=cdr$CDR_VERSION
+echo "Making big query dataset for cloudsql cdr"
```

Comment:
Add "public" to this message and the ones below?

Reply:
fixed. pushed. rebased. Regarding the import. I closed that one as its bigger than this one and this was more related and finishing up previous. IE only bq dumps . Next one will address a new goal making mysqld databases.